voss/harness/tools.py

```
@@ -200,15 +206,40 @@ async def memory_recall(query: str, top_k: int = 5, source: str | None = None) -
+        # a `global:` locator prefix so they survive RRF dedup distinctly and
+        # render with a `[global]` label (stripped from the displayed locator).
+        if global_store is not None:
+            try:
+                global_hits = global_store.recall(query, top_k=top_k, source=source)
+                if global_hits:
+                    import dataclasses as _dc
+
+                    tagged = [_dc.replace(h, locator=f"global:{h.locator}") for h in global_hits]
+                    hits = MemoryStore._rrf_merge([hits, tagged], top_k=top_k)
+            except Exception:  # noqa: BLE001 — global recall must not break the turn
+                pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)